File: src/ambil_komentar.php
<?php
    $con = mysqli_connect("localhost","root","","progin_405_13510003");
    
    if (mysqli_connect_errno($con)) {
        echo "Gagal Koneksi";
    } else {
        $id_tugas = $_REQUEST["id_tugas"];
        $mulai = 10 * ($_REQUEST["start"]-1);
    
        if ($stmt = mysqli_prepare($con, "SELECT * FROM komentar WHERE (id_tugas = ?) ORDER BY waktu LIMIT ?,10")) {
            mysqli_stmt_bind_param($stmt,"dd",$id_tugas,$mulai);
            mysqli_stmt_execute($stmt);
            $h1="";
            $h2=0;
            $h3="";
            $h4="";
            mysqli_stmt_bind_result($stmt,$h1,$h2,$h3,$h4);
            
            $array=array();
            
            while (mysqli_stmt_fetch($stmt)) {
                $array[] = array($h1,$h3,$h4);
            }
            
            mysqli_stmt_close($stmt);
            
            foreach ($array as $row) {
                $result = mysqli_query($con,"SELECT avatar FROM pengguna WHERE (username = '$row[0]')");
                $avatar = mysqli_fetch_row($result);
                echo $row[0],"\n",$row[1],"\n",$row[2],"\n",$avatar[0],"\n";
                mysqli_free_result($result);
            }
        } else {
            echo "Gagal mengambil komentar";
        }
        mysqli_close($con);
    }
?>

File: Tubes3/src/java/RincianTugas/ambil_komentar.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package RincianTugas;

import java.sql.*;
import java.io.IOException;
import java.io.PrintWriter;
import java.util.logging.Level;
import java.util.logging.Logger;
import javax.servlet.ServletException;
import javax.servlet.annotation.WebServlet;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

/**
 *
 * @author Christianto
 */
@WebServlet(name = "ambil_komentar", urlPatterns = {"/ambil_komentar"})
public class ambil_komentar extends HttpServlet {
    private Connection conn;
    private Statement query;
    
    /**
     * Processes requests for both HTTP
     * <code>GET</code> and
     * <code>POST</code> methods.
     *
     * @param request servlet request
     * @param response servlet response
     * @throws ServletException if a servlet-specific error occurs
     * @throws IOException if an I/O error occurs
     */
    protected void processRequest(HttpServletRequest request, HttpServletResponse response)
            throws ServletException, IOException {
        response.setContentType("text/html;charset=UTF-8");
        PrintWriter out = response.getWriter();
        try {
            /* TODO output your page here. You may use following sample code. */
            Class.forName("com.mysql.jdbc.Driver");
            conn = DriverManager.getConnection("jdbc:mysql://localhost:3306/progin_405_13510003", "root", "");
            query = conn.createStatement();
            ResultSet result = query.executeQuery("SELECT * FROM komentar "
                    + "WHERE (id_tugas = "+request.getParameter("id_tugas")+") ORDER BY waktu LIMIT "
                    + (Integer.parseInt(request.getParameter("start"))-1)*10+",10");
            
            result.last();
            String[] username = new String[result.getRow()];
            String[] waktu = new String[result.getRow()];
            String[] isi = new String[result.getRow()];
            result.beforeFirst();

            for (int i=0;result.next();++i) {
                username[i] = result.getString("username");
                waktu[i] = result.getString("waktu");
                isi[i] = result.getString("isi");
            }
            result.close();
            
            for (int i=0;i<username.length;++i) {
                ResultSet r2 = query.executeQuery("SELECT avatar FROM pengguna WHERE (username = '"
                        +username[i]+"')");
                out.print(username[i]+"\n");
                out.print(waktu[i]+"\n");
                out.print(isi[i]+"\n");
                r2.first();
                out.print(r2.getString(1)+"\n");
                r2.close();
            }
        } catch (ClassNotFoundException ex) {
            out.println("Failed to create connection");
        } catch (SQLException ex) {
            out.println("Failed to execute SQL query");
        } finally {            
            out.close();
        }
    }

    // <editor-fold defaultstate="collapsed" desc="HttpServlet methods. Click on the + sign on the left to edit the code.">
    /**
     * Handles the HTTP
     * <code>GET</code> method.
     *
     * @param request servlet request
     * @param response servlet response
     * @throws ServletException if a servlet-specific error occurs
     * @throws IOException if an I/O error occurs
     */
    @Override
    protected void doGet(HttpServletRequest request, HttpServletResponse response)
            throws ServletException, IOException {
        processRequest(request, response);
    }

    /**
     * Handles the HTTP
     * <code>POST</code> method.
     *
     * @param request servlet request
     * @param response servlet response
     * @throws ServletException if a servlet-specific error occurs
     * @throws IOException if an I/O error occurs
     */
    @Override
    protected void doPost(HttpServletRequest request, HttpServletResponse response)
            throws ServletException, IOException {
        processRequest(request, response);
    }

    /**
     * Returns a short description of the servlet.
     *
     * @return a String containing servlet description
     */
    @Override
    public String getServletInfo() {
        return "Short description";
    }// </editor-fold>
}
